generated/nidaqmx/nidaqmx.proto

```diff
@@ -1699,34 +1699,19 @@ enum DeviceBoolAttribute {
 enum DeviceInt32Attribute {
   DEVICE_INT32_ATTRIBUTE_UNSPECIFIED = 0;
   DEVICE_ATTRIBUTE_BUS_TYPE = 8998;
-  DEVICE_ATTRIBUTE_PRODUCT_CATEGORY = 10665;
-  DEVICE_ATTRIBUTE_CI_CURRENT_UTC_OFFSET = 12320;
-  DEVICE_ATTRIBUTE_NAV_MODE = 12325;
-  DEVICE_ATTRIBUTE_NAV_ALT_REF = 12329;
-  DEVICE_ATTRIBUTE_NAV_ANT_STATUS = 12334;
-  DEVICE_ATTRIBUTE_NAV_CURRENT_UTC_OFFSET = 12336;
-}
-
-enum DeviceInt32ArrayAttribute {
-  DEVICE_INT32_ARRAY_ATTRIBUTE_UNSPECIFIED = 0;
   DEVICE_ATTRIBUTE_AI_TRIG_USAGE = 10630;
   DEVICE_ATTRIBUTE_AO_TRIG_USAGE = 10631;
   DEVICE_ATTRIBUTE_DI_TRIG_USAGE = 10632;
   DEVICE_ATTRIBUTE_DO_TRIG_USAGE = 10633;
   DEVICE_ATTRIBUTE_CI_TRIG_USAGE = 10634;
   DEVICE_ATTRIBUTE_CO_TRIG_USAGE = 10635;
   DEVICE_ATTRIBUTE_AI_COUPLINGS = 10644;
-  DEVICE_ATTRIBUTE_AI_SUPPORTED_MEAS_TYPES = 12242;
-  DEVICE_ATTRIBUTE_AO_SUPPORTED_OUTPUT_TYPES = 12243;
-  DEVICE_ATTRIBUTE_CI_SUPPORTED_MEAS_TYPES = 12244;
-  DEVICE_ATTRIBUTE_CO_SUPPORTED_OUTPUT_TYPES = 12245;
-  DEVICE_ATTRIBUTE_AI_SAMP_MODES = 12252;
-  DEVICE_ATTRIBUTE_AO_SAMP_MODES = 12253;
-  DEVICE_ATTRIBUTE_CI_SAMP_MODES = 12254;
-  DEVICE_ATTRIBUTE_CO_SAMP_MODES = 12255;
-  DEVICE_ATTRIBUTE_NAV_SUPPORTED_MEAS_TYPES = 12323;
-  DEVICE_ATTRIBUTE_NAV_TRIG_USAGE = 12324;
-  DEVICE_ATTRIBUTE_AI_DIG_FLTR_TYPES = 12551;
+  DEVICE_ATTRIBUTE_PRODUCT_CATEGORY = 10665;
+  DEVICE_ATTRIBUTE_CI_CURRENT_UTC_OFFSET = 12320;
+  DEVICE_ATTRIBUTE_NAV_MODE = 12325;
+  DEVICE_ATTRIBUTE_NAV_ALT_REF = 12329;
+  DEVICE_ATTRIBUTE_NAV_ANT_STATUS = 12334;
+  DEVICE_ATTRIBUTE_NAV_CURRENT_UTC_OFFSET = 12336;
```

Comment:
FYI - I don't think these were ever officially released or used in any products, but its not a big deal.